index.html

```diff
@@ -4,15 +4,264 @@
+            has been the 
+          </p>
+        </li>
+        <li class="workshops-items">
+          <img src="icons/event2.png" alt="Workshop icon"/>
+          <h4>Producer's Seminar</h4>
+          <p>
+            EDM Producers share their experience 
+            Is simply dummy text of the printing  
+            has been the 
+          </p>
+        </li>
+        <li class="workshops-items">
+          <img src="icons/event3.png" alt="Workshop icon"/>
+          <h4>Social Media</h4>
+          <p>
+            Media experts share their knowledge  
+            and typesetting industry. Lorem Ipsum 
+            has been 
+          </p>
+        </li>
+        <li class="workshops-items">
+          <img src="icons/event4.png" alt="Workshop icon"/>
+          <h4>Lounge set</h4>
+          <p>
+            Above & beyond launches ABGT 600 
+            and typesetting industry. Lorem Ipsum 
+            has been the
+          </p>
+        </li>
+        <li class="workshops-items">
+          <img src="icons/event5.png" alt="Workshop icon"/>
+          <h4>Tiesto's case</h4>
+          <p>
+            How the world's most famous DJ made it 
+            Is simply dummy text of the printing 
+            has been 
+          </p>
+        </li>
+      </ul>
+      <div class="see-program">
+        <h4>SEE THE WHOLE PROGRAM</h4>
+      </div>
+      <button class="join" type="button">
+        Join ECM 2022 now
+      </button>
+    </section> <!-- END OF MAIN -->
+
+    <section class="talks" id="Talks"> <!-- START OF TALKS -->
+      <div class="h3-header">
+        <h3>EDM Professionals</h3>
+        <div class="dash">  </div>
+      </div>
+      <ul class="speakers" id="Speakers">
+        
+        <!-- HERE GO THE DYNAMIC CARDS -->
+
+      </ul>
+      <div class="more">
+        <button class="more-speakers" type="button">
+          MORE
+          <img src="icons/arrow-down.png" alt="">
+        </button>
+      </div>
+    </section> <!-- END OF TALKS -->
+
+    <section class="about" id="About">  <!-- START OF ABOUT -->
+      <span>
+        "Welcome, Electronic music lovers!"
+      </span>
+      <h2>
+        EDM MARKETING <br>
+        CONFERENCE 2022
+      </h2>
+      <p>
+        “Since its inception more than 30 years ago, 
+        EMC and DJ Expo have always maintained 
+        a deep connection to the DJ world and we’ve 
+        always looked to broaden its reach,” she said. 
+        “We believe the re-branding, which coincides 
+        with the show’s exciting move to the Hard 
+        Rock, will can help in that endeavor. So to 
+        borrow our New York motto… Excelsior – ever 
+        upwardI.
+      </p>
+      <p class="center-p">
+        Please contact us per email for any further 
+        questions about EDM Marketing Conference 2022! <br>
+        <span class="email-contact">
+          emc2022@emcbrazil.org
+        </span>
+      </p>
+    </section> <!-- END OF ABOUT -->
```

Comment:
- [ ] Great work job far, you have done a big job, but I think you should make this about section be a page, not a section, it should be a single page separated from the home page, please create another about.html page for it.
- [ ] make sure your header contains navigation links (`about, home`) to navigate from `about page` to `home page` and the reverse too.
- [ ] Your logo should be a link to navigate to the home page